doc/md/fundamentals/2-types/4-tuples.md

```diff
@@ -44,17 +44,17 @@ let second = ghost.1; // 25
 
 Tuples are useful for returning multiple values from a function without requiring a separate data structure.
 
-```motoko no-repl
+```motoko
 func getUserInfo() : (Text, Nat) {
     ("Ghost", 30);
-}
+};
 
 getUserInfo();
 ```
 
 ## Nesting tuples
 
-Tuples can be used to represent coordinate pairs, allowing for structured calculations such as finding the gradient of a line. The gradient (or slope) between two points `$(x_1, y_1)$` and `$(x_2, y_2)$` is calculated using the formula:
+Tuples can be used to represent coordinate pairs, allowing for structured calculations such as finding the gradient of a line. The gradient (or slope) between two points $(x_1, y_1)$ and $(x_2, y_2)$ is calculated using the formula:
```

Comment:
I think the following example might be clearer with a couple of type definitions:

```
//  Point is a tuple
type Point = (Float, Float);
// Line is a tuple of points, that is, a nested tuple
type Line = (Point, Point);
func calculateGradient(line: Line) : ?Float {
let ((x1, y1), (x2, y2)) = line;  
  if (x1 == x2) {  
    null // Gradient is undefined for a vertical line  
  } else {  
    ?((y2 - y1) / (x2 - x1)) // Wraps the result as an option  
  }  
}  
let line : Line = ((2.0, 3.0), (5.0, 7.0));
calculateGradient(line); // should show the gradient when run
```